sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -52,14 +50,19 @@ def __init__(self, endpoint, credential, **kwargs):
         self._credential = credential
         super(ContainerRegistryClient, self).__init__(endpoint=endpoint, credential=credential, **kwargs)
 
+    def _get_digest_from_tag(self, repository, tag):
+        # type: (str, str) -> str
+        tag_props = self.get_tag(repository, tag)
+        return tag_props.digest
```

Comment:
Do we need to add a comment to the docstrings that if a tag is supplied instead of a digest, an extra service call needs to be made?
Could be worth checking with the crew if they feel customers should be aware of that distinction. Maybe not, as I can't see this lib being particular perf-dependent...

Reply:
We are thinking about this and whether we want to transition to strictly taking a digest where digests are needed by the service. If we go with this decision this method will be removed entirely. If not, I will add a comment to the docstrings where this method is used.